### PR TITLE
Custom UI on layout, homepage, admin, show,list user,book

### PR DESCRIPTION
**Pull request by ngotungson**

### CODE REVIEW

**Comment by phanbt58** [comment]
okie

File: app/assets/javascripts/custom.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$( document ).ready(function() {
+  $("#messagebox").delay(5000).fadeOut(300)
```
Comment (nhunght) [inline app/assets/javascripts/custom.js]: nháy đơn

File: app/assets/javascripts/custom.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$( document ).ready(function() {
```
Comment (nhunght) [inline app/assets/javascripts/custom.js]: $(document).ready(function()

File: app/uploaders/book_image_uploader.rb [hunk 3]
```diff
@@ -34,7 +34,7 @@ def store_dir
 
   # Create different versions of your uploaded files:
   version :thumb do
-    process :resize_to_fit => [300, 400]
+    process :resize_to_fit => [150, 200]
```
Comment (nhunght) [inline app/uploaders/book_image_uploader.rb]: resize_to_fit: [150, 200]

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+
+<ul class="list-group">
   <% @users.each do |user| %>
-    <li>
-      <%= link_to user.name, user %>
-      <%= user.email %>
-      <%= user.avatar %>
-    </li>
+  <li class="list-group-item">
+  <span> <%= image_tag user.avatar.url(:thumb), size: "50x50" if user.avatar %></span>
```
Comment (nhunght) [inline app/views/users/index.html.erb]: indent

**Pull request merged**